stp_zmq/zstack.py

```diff
@@ -359,6 +365,9 @@ def open(self):
         )
 
     def close(self):
+        if self.listener_monitor is not None:
+            self.listener.disable_monitor()
```

Comment:
Disabling monitor on the non-listener socket is done differently. Is it intended?

Reply:
See comment above.